fastautils/io/file_loader.py

```diff
@@ -0,0 +1,38 @@
+from typing import List
+
+import numpy as np
+
+from ..core import Sequence
+
+# TODO: Add support fot other fasta file type
```

Comment:
I think there is a typo here. fot --> for

Reply:
Cierto

Reply:
Ya quedó